src/Model/Scope/Condition.php

```diff
@@ -195,8 +195,16 @@ public function toQueryArguments(): array
                     }
 
                     foreach (array_reverse($refModels) as $refModel) {
-                        if ($field === '#') {
-                            $field = $value ? $refModel->action('count') : $refModel->action('exists');
+                        // # is alias for count(*)
+                        if ($field === '#' || $field === 'count(*)') {
+                            $field = $refModel->action($value ? 'count' : 'exists');
```

Comment:
If operator is `<`?

Reply:
then we should probably optimize common exprs can we evaluated to exists

better (much better for performance) solution would be if we can generated queries like:
```
select count(*) as c from (select <standard query> limit X (<our value> + 1)) t having c <our operator> <cond value>
```